modules/relaychain-interface/src/lib.rs

```diff
@@ -16,7 +16,13 @@
 // You should have received a copy of the GNU General Public License
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
-//! Homa xcm module.
+//! Relaychain Interface module.
```

Comment:
This module is for all XCM related functions, instead of only for relay chain. Better to rename with `XCM` in it.